### PR TITLE
azure js task 3

### PR DESCRIPTION
**Pull request by nuquerna**

### CODE REVIEW

**Comment by apetrik** [comment]
Could you please add urls or some other evidence that your api is up and running and that it's integrated with your static website

File: http-get-product/function.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get",
+        "post"
```
Comment (apetrik) [inline http-get-product/function.json:10]: do you need a post method here for now?

File: http-get-product/index.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
```
Comment (apetrik) [inline http-get-product/index.ts]: looks like this is just an example function. I guess you can delete it

Comment (nuquerna) [reply http-get-product/index.ts]: Removed example

**Comment by nuquerna** [comment]
> Could you please add urls or some other evidence that your api is up and running and that it's integrated with your static website

https://rg-product-service-sand-ne-0024.azure-api.net/products
https://rg-product-service-sand-ne-0024.azure-api.net/products/3

**Pull request merged**